src/main/java/com/treasure/hunt/game/GameHistory.java

```diff
@@ -1,5 +1,11 @@
 package com.treasure.hunt.game;
 
-public class GameHistory {
+import com.treasure.hunt.strategy.geom.GeometryItem;
 
+import java.util.List;
+
+// TODO add any useful code
+class GameHistory {
+    void dump(List<GeometryItem> list) {
+    }
```

Comment:
Doesn't this piece require an rx Observable? Where will it be used?

Reply:
I'll add this of course

Reply:
Thing for another branch i suppose.